HW_2/task_2.py

```diff
@@ -0,0 +1,4 @@
+Eleks = ["Michael", "Alexander", "Paula"]
+Toshiba = ["Alexander", "Naomi", "Ginger"]
+Toshiba = Toshiba.append(Eleks)
```

Comment:
naming. Even if it is personal names your variables should be from the small letters. 

Also after your manipulations eleks still has employees. Please fix it